tests/benchmarks/test_rechunk.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import dask.array as da
+import pytest
+
+from typing import Any
+
+import dask
+from ..utils_test import cluster_memory, scaled_array_shape
+
+@pytest.fixture(params=["8 MiB", "128 MiB"])
+def chunksize(request):
+    return request.param
+
+
+@pytest.fixture
+def configure_chunksize(chunksize):
+    with dask.config.set({"array.chunk-size": chunksize}):
+        yield
+
+def test_tiles_to_rows(small_client, memory_multiplier, configure_chunksize, configure_rechunking):
+    memory = cluster_memory(small_client)
+    shape = scaled_array_shape(memory * memory_multiplier, ("x", "x"))
+
+    rng = da.random.default_rng()
```

Comment:
I'm curious, why this rather than the normal `da.random.random` function?

Reply:
Originally because of coiled/benchmarks#790, but it looks like dask/dask#10247 may have fixed this.